src/components/QuestDetails/components/Reward/Reward.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import RewardImage from '@/assets/RewardImage.png'
+
+import { QuestReward } from '../../types'
+import Reward from './index'
+
+const meta: Meta<typeof Reward> = {
+  title: 'Quests/Reward',
```

Comment:
should we change this to `Quests/QuestDetails/StreakProgress`? same as StreakProgress since it is only used by the quest details